jspdf.js

```diff
@@ -1015,7 +1015,12 @@ var jsPDF = (function(global) {
                 return API.output('dataurlnewwindow');
               }
             }
-            saveAs(getBlob(), options);
+            let filesaver_status = saveAs(getBlob(), options);
+            return new Promise((resolve, reject)=>{
```

Comment:
maybe better checking if browser is ECMA 6 valid for compatibility reasons and use old method for older browsers and new for newer